outputs.tf

```
@@ -17,3 +17,8 @@ output "task_definition_family" {
   value       = module.service.task_definition_family
   description = "ECS task definition family"
 }
+
+output "alb_listener_arn" {
+  description = "The ARN of the ALB listener."
+  value       = aws_lb_listener.app[0].arn
+}
```

The ALB listeners are using count and might be 0 or more.

This would most likely create a bug if the `local.enabled` is `false` or if `var.alb_listeners` is empty.

I would rather output it as a list and then get the first element of that list (if any) where you need it.

```suggestion
output "alb_listener_arn" {
  description = "The ARN of the ALB listeners."
  value       = aws_lb_listener.app.*.arn
}
```

Note that I did not test that code.